app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import NotVaccinatedError
+from app.errors import OutdatedVaccineError
+from app.errors import NotWearingMaskError
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
+    has_mask = True
+    without_mask = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except NotVaccinatedError:
+            return "All friends should be vaccinated"
+        except OutdatedVaccineError:
+            return "All friends should be vaccinated"
```

Comment:
You can unite these to one `exception` block, because `NotVaccinatedError` and `OutdatedVaccineError` have the same parent class
```suggestion
        except VaccineError:
            return "All friends should be vaccinated"
```